TLM/TLM/Custom/PathFinding/CustomPathFind.cs

```diff
@@ -3979,10 +3979,22 @@ private struct BufferItem {
         }
 
 #if VEHICLERESTRICTIONS
+        /// <summary>
+        /// Check if lane can be used by vehicle requesting path
+        /// </summary>
+        /// <param name="segmentId">segment ID</param>
+        /// <param name="segmentInfo">">segment NetInfo</param>
+        /// <param name="laneIndex">lane index</param>
+        /// <param name="laneInfo">lane LaneInfo</param>
+        /// <param name="laneId">lane ID</param>
+        /// <returns>true if vehicle restrictions is disabled,
+        /// lane vehicle restriction allow vehicle type or lane is start/end of the path
+        /// otherwise false</returns>
         private bool CanUseLane(ushort segmentId,
                                 NetInfo segmentInfo,
                                 int laneIndex,
-                                NetInfo.Lane laneInfo) {
+                                NetInfo.Lane laneInfo,
+                                uint laneId) {
```

Comment:
Now that we're passing in `laneId`, do we stil need the `laneInfo`?  Would it be better to use the `laneId.ToLane()` extension?

Or are those referring to different lanes? If so the xmldoc and param names don't make that obvious.

Reply:
It's not `NetLane` struct ("instance" of lane) but `NetInfo.Lane` ("definition" of segment lane), see at `3536`